src/components/Receiving/ItemDetails.js

```diff
@@ -234,6 +236,7 @@ class ItemDetails extends Component {
     const poLineNumber = get(lineItems, [poLineId, 0, 'poLineNumber'], '');
     const title = get(lineItems, [poLineId, 0, 'title'], '');
     const isReviewScreen = currentLine >= poLineIdsList.length;
+    const checkedItemsList = sortBy(lineItems.reviewDetails, ['title', 'locationId']);
```

Comment:
sorting in render? does it make sense to sort items in `check` handler?

Reply:
fixed